server/channels/store/storetest/shared_channel_store.go

```diff
@@ -699,6 +699,19 @@ func testGetSharedChannelRemotes(t *testing.T, rctx request.CTX, ss store.Store)
 		}
 	})
 
+	t.Run("Get only unconfirmed shared channel remotes for remote", func(t *testing.T) {
+		opts := model.SharedChannelRemoteFilterOpts{
+			RemoteId:        remoteId,
+			OnlyUnconfirmed: true,
```

Comment:
I guess `OnlyUnconfirmed` implies that `IncludeUnconfirmed` is true, even though it is false.  I wonder if a `ExcludeConfirmed` would be easier to reason about and would achieve the same thing,

Reply:
Done at https://github.com/mattermost/mattermost/pull/28176/commits/c281c1e4f832689dd4422c0a0c362d2a6b5884a5, PTAL